tasks/sonar_runner.js

```diff
@@ -44,7 +44,10 @@ module.exports = function (grunt) {
         var dryRun = options.dryRun;
         var done = this.async();
 
-        options.sonar.language = options.sonar.language || 'js';
```

Comment:
In commit 4bb9ea434e75459bd624d995814c4e64ec845a4d I updated the documentation. Is this enough or should I change it?